lib/net/smtp.rb

```diff
@@ -831,6 +862,16 @@ def open_message_stream(from_addr, *to_addrs, &block)   # :yield: stream
 
     DEFAULT_AUTH_TYPE = :plain
 
+    # call-seq:
+    #   authenticate(username, secret, type = DEFAULT_AUTH_TYPE, **, &)
+    #
```

Comment:
This change isn't accurate for the current release.  It should be removed from the PR and moved to #71.
```suggestion
```